src/App.vue

```diff
@@ -1,17 +1,20 @@
 <script setup lang="ts">
   import { RouterView } from 'vue-router'
   import TheHeader from '@/components/nav/TheHeader.vue'
+  import TheFooter from './components/footer/TheFooter.vue'
```

Comment:
Wollen wir Header und Footer in einen Folder "layout" verschieben? Dann brauchen wir nicht für beides jeweils einen Folder.

Reply:
Done.